robot-name/Pratster95/robot_name.go

```diff
@@ -0,0 +1,40 @@
+package robotname
+
+import (
+	"fmt"
+	"math/rand"
+)
+
+// Define the Robot type here.
+type Robot struct {
+	name string
+}
+
+var nameUsed = map[string]bool{}
+
+const max = 26 * 26 * 10 * 10 * 10
+
+func (r *Robot) Name() (string, error) {
+	if r.name != "" {
+		return r.name, nil
+	}
+	if len(nameUsed) >= max {
+		return "", fmt.Errorf("No Names left")
```

Comment:
Errors in go should start with lowercase.
```suggestion
		return "", fmt.Errorf("no names left")
```